docs/06-concepts/11-authentication/04-providers/06-custom-providers.md

```diff
@@ -113,25 +113,53 @@ class MyAuthenticationEndpoint extends Endpoint {
 
 The example above shows how to create an `AuthenticationResponse` with the auth token and user information.
 
-### Remove auth token
+## Managing Authentication Keys and Signing Out Users
```

Comment:
Suggestion: Shorten to `Revoking authentication keys`. 

Check rest of the documentation but I think we only use a capital letter for the first letter in the headers.

Please fix this for the full PR.